src/main/java/seedu/address/logic/commands/SearchPublicAddressCommand.java

```diff
@@ -79,16 +89,22 @@ public CommandResult execute(Model model) throws CommandException {
      * {@code personToEdit}.
      */
     private String generateSuccessMessage(List<Person> personsWithPublicAddressMatch) {
```

Comment:
This logic below makes more sense to be inside execute,  the name implies that it will generate success message but sometimes it may throw out not found messages. Bring the check outside into execute would be better

Reply:
yes sir, astude observation

Reply:
that's resolved